code/modules/clothing/gloves/color.dm

```diff
@@ -85,6 +85,14 @@
 	. = ..()
 	siemens_coefficient = pick(0,0.5,0.5,0.5,0.5,0.75,1.5)
 
+	if(Holiday == APRIL_FOOLS_DAY && siemens_coefficient != 0)
+		name = "insulted gloves"
+		desc = "These gloves will mock the wearer when shocked."
+
+/obj/item/clothing/gloves/fyellow/insulted
```

Comment:
It's possible for the siemens_coefficient to be 0 with these, so you may want to consider the following options:
1) Make a new pick() to replace it. In this case, you could make something like:
```
/obj/item/clothing/gloves/fyellow/insulted/New()
   . = ..()
   if(siemens_coefficient == 0) //never safe with these.
      siemens_coefficient = pick(0.5,0.5,0.5,0.5,0.75,1.5)
```
2) Or, leave it in. You might get The Lucky Ones. Note the comment in the other file regarding messaging with these gloves if they roll a 0. Also note if you go this route, you can likely remove the siemens_coefficient from the april fools day check above and be fine.